lib/metanorma/collectionconfig/collectionconfig.rb

```diff
@@ -0,0 +1,238 @@
+require "fileutils"
+require "relaton"
+require "relaton-cli"
+require "yaml"
+require "shale"
+require "shale/adapter/nokogiri"
+require_relative "shale_monkeypatch"
+require_relative "helper"
+
+module CollectionConfig
+  ::Shale.xml_adapter = ::Shale::Adapter::Nokogiri
+
+  # TODO sidesteps bibdata redefinition of filter, will remove
+  class Manifest < ::Shale::Mapper
+    class ::Array
+      alias filter select
+    end
```

Comment:
Why do we need this? Why a monkey patch when it is unnecessary?

Reply:
it is not unnecessary, I explicitly said why it is there. relaton-iso has monkeypatched filter, and until that is removed, it is causing Shale's use of filter to crash.